src/Dobble/Card.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Marmelab\Dobble;
+
+class Card implements \Countable
+{
+    /**
+     * @todo DobbleSymbol class
+     */
+    private $symbols = [];
+
+    public function __construct(array $symbols)
+    {
+        $this->symbols = $symbols;
+
+        // The deck validation uses the fonction "array_unique" to check
+        // if some cards have same values, so we need to sort symbols
+        // for avoid that same values seems differents.
+        asort($this->symbols);
```

Comment:
The deck need cards with sorted symbols but the cards can be in this state before, I think it's more elegant to display an array which is already sorted.
However, the `$symbols` attribute is private.

Is it a good reason or there is a better solution ?